dev/tests/api-functional/testsuite/Magento/GraphQl/CatalogUrlRewrite/UrlResolverTest.php

```diff
@@ -219,12 +219,10 @@ public function testRedirectsAndCustomInput()
         $urlRewriteModel->setRedirectType('301');
         $urlRewriteModel->setId($urlRewriteModel->getId());
         $urlRewriteModel->save();
-
-        ObjectManager::getInstance()->get(\Magento\TestFramework\Helper\CacheCleaner::class)->cleanAll();
         //modifying query by adding spaces to avoid getting cached values.
         $this->queryUrlAndAssertResponse(
             (int) $product->getEntityId(),
-            $customUrl,
+            $customUrl . ' ',
```

Comment:
It was added as described in comment https://github.com/magento/magento2/pull/31053/files/9ec34d8ca0a27ba9e34d8ad4803b6217175605d9#diff-e996da3d86e3da3160dd34c221e07fd9c45874f3ea3795b574e7e09ed004fad3R222
Since earlier we use `CacheCleaner::cleanAll()` and we got not cached response.